deploy/vagrant/README.md

```diff
@@ -1,5 +1,36 @@
 This doc is for contributors on deploy module, for users, please go to the [online doc](http://tachyon-project.org/master/Deploy-Module.html).
 
+## Tools
+
+### Vagrant
+
+[Vagrant](https://www.vagrantup.com), here is the [download link](https://www.vagrantup.com/downloads.html).
+
+#### What is Vagrant?
+
+Vagrant can create VM images (VirtualBox, VMWare Fusion), Docker containers, and AWS and OpenStack
+instances.
+
```

Comment:
I think this section could be better as

```### Vagrant```
```[Vagrant](https://www.vagrantup.com) can create ...```

Basically merge the ```What is Vagrant``` portion into the description.